paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodEmbeddedLayoutUIScreenshotTest.kt

```diff
@@ -47,7 +47,7 @@ class PaymentMethodEmbeddedLayoutUIScreenshotTest {
                 onSelectSavedPaymentMethod = {},
                 onManageOneSavedPaymentMethod = {},
```

Comment:
I was actually thinking a bit about the reverse. I thought all of the fields outside of row style were the same. And row style was the only "difference".

Reply:
How about both? Added `TestPaymentMethodLayoutUi` helper function